account_statement_import_camt/views/account_bank_statement_import.xml

```diff
@@ -11,6 +11,11 @@
                 <li>CAMT</li>
                 <li>zipped CAMT</li>
             </ul>
+            <ul id="statement_format" position="after">
+                <p
+                    id="checking_name_unique"
+                >Unique bank statement name checking can be turned on at Settings > Invoicing > Bank Statement Import > Bank Statement Import.</p>
```

Comment:
```suggestion
                >Unique bank statement name checking for CAMT statements can be turned on at Settings > Invoicing > Bank Statement Import > Bank Statement Import.</p>
```

Reply:
Thanks for the suggestion. It has been applied with a little difference.